src/apify_client.ts

```diff
@@ -53,6 +53,7 @@ export class ApifyClient {
             requestInterceptors: ow.optional.array,
             timeoutSecs: ow.optional.number,
             token: ow.optional.string,
+            userAgentSuffix: ow.any(ow.optional.string, ow.optional.array.ofType(ow.string)),
```

Comment:
Assertion doesn't match type

Reply:
I swear I started with modifying the type and only then modified this, how come the changes were lost :D